src/main/java/ch/njol/skript/expressions/ExprExplosionBlockYield.java

```diff
@@ -74,19 +74,24 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 	@Override
 	@Nullable
 	public Class<?>[] acceptChange(final ChangeMode mode) {
-		if (mode == ChangeMode.REMOVE_ALL || mode == ChangeMode.RESET)
-			return null;
-		return CollectionUtils.array(Number.class);
+		switch (mode) {
```

Comment:
What's the point of making a switch statement when you're only checking for the delete mode?